src/assets/stylesheets/hub-create.scss

```diff
@@ -194,7 +196,17 @@
 	      @media (max-width: 520px) {
 		      display: none;
```

Comment:
admittedly there's not a whole lot of space, but it's a bit odd to see that at mobile widths the author is not credited

can this style be removed to ensure the author's name is always visible?

<img width="1283" alt="screen shot 2018-04-25 at 6 34 51 pm" src="https://user-images.githubusercontent.com/203725/39281045-80932aaa-48b7-11e8-8539-2fa39c557af7.png">
<img width="846" alt="screen shot 2018-04-25 at 6 34 41 pm" src="https://user-images.githubusercontent.com/203725/39281046-80a84214-48b7-11e8-867f-199314b27e5f.png">
<img width="603" alt="screen shot 2018-04-25 at 6 34 32 pm" src="https://user-images.githubusercontent.com/203725/39281047-80bc5b5a-48b7-11e8-8fb8-caa31223d7ff.png">